abis_mapping/templates/incidental_occurrence_data_v3/templates/instructions.md

```diff
@@ -68,6 +68,17 @@ datatype format, and examples.</ins>
 
 {{tables.fields}}
 
+## CHANGELOG
+
+Changes from Incidental Occurrence Data Template v2.0.0
+
+### CHANGED FIELDS
+
+* `conservationJurisdiction` is renamed to [`conservationAuthority`](#conservationAuthority-field).
```

Comment:
For now yes but i think the intent is that a new concept gets created as part of ABIS/ausbigg working group(s). Will raise a ticket for follow up.